samcli/lib/telemetry/event.py

```diff
@@ -144,7 +150,11 @@ class EventTracker:
 
     @staticmethod
     def track_event(
-        event_name: str, event_value: str, session_id: Optional[str] = None, exception_name: Optional[str] = None
+        event_name: str,
+        event_value: str,
+        session_id: Optional[str] = None,
+        thread_id: Optional[UUID] = None,
```

Comment:
does putting `thread_id` before `exception_name` as positional argument affect any existing `track_event` call?

Reply:
Nope it does not, just verified in the code base. The only call right now that makes use of the `session_id` property is the call in `local_apigw_service.py` on line 701, which calls it by keyword.